README.md

```diff
@@ -97,6 +118,43 @@ The plugin also supports the following Teradata connection parameters:
 
 For full description of the connection parameters see https://github.com/Teradata/python-driver#connection-parameters.
 
+### Stored Password Protection
```

Comment:
I would move this section directly under logmech as they are all related to authentication.

Reply:
Moved Stored Password Protection under logdata